gulpfile.js

```diff
@@ -10,155 +10,169 @@ var gulpSequence = require('gulp-sequence');
 var _ = require('underscore');
 var browserSync = require('browser-sync').create();
 
-var books = ["ukrainian", "physics", "maths", "biology", "english", "geography", "history", "law", "english2", "anticorruption-lesson"];
+var books = ["ukrainian", "physics", "maths", "biology", "english", "geography", "history", "law", "english2", "anticorruption-lesson", "vaccination"];
 
 var lessons = ["1", "2"];
 
```

Comment:
for some reason command `gulp` doesn't work alone anymore, we need to call `gulp templates` before. Can you check what changed? Ideally `gulp` should do the job

Reply:
A lot of problems was caused by my poor gulp knowledge.
But there was `[error](https://stackoverflow.com/questions/36897877/gulp-error-the-following-tasks-did-not-complete-did-you-forget-to-signal-async)` running gulpfile that I fixed in last commit.

Reply:
nice!